database/src/main/java/com/firebase/ui/database/FirebaseListAdapter.java

```diff
@@ -27,21 +29,21 @@
 public abstract class FirebaseListAdapter<T> extends BaseAdapter implements FirebaseAdapter<T> {
     private static final String TAG = "FirebaseListAdapter";
 
-    protected final Activity mActivity;
+    protected final Context mContext;
```

Comment:
Can we remove the DB changes since they aren't related to this PR? Otherwise LGTM! 😀